spec/lib/myfinance/request_spec.rb

```diff
@@ -19,5 +19,11 @@
       expect_any_instance_of(Typhoeus::Request).to receive(:response)
       subject.run
     end
+
+    it 'generates the base 64 of the user token' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.